docker-vms/http_api/api_tokens.yaml

```diff
@@ -0,0 +1,5 @@
+# Example token. Replace for production
+-
+  account: "9e60a884-4b59-44db-a8a4-52c4bb02a874"
+  conversation: "2742b45a-ef7c-43f9-b782-5af8c4cc7e7a"
+  tokens: ["563735b0ea62"]
```

Comment:
Hmm. Shouldn't the account and authentication tokens also be set via environment variables?